src/components/tiles/text/text-tile.tsx

```diff
@@ -222,6 +222,7 @@ export default class TextToolComponent extends BaseComponent<ITileProps, IState>
                 onFocus={this.handleFocus}
                 onBlur={this.handleBlur}
                 className={`ccrte-editor slate-editor ${slateClasses || ""}`}
+                data-testid="question-tile-placeholder-text"
```

Comment:
This doesn't seem to be used.
I would remove it since the name isn't accurate anyway.